src/workerd/util/BUILD.bazel

```diff
@@ -59,6 +55,36 @@ wd_cc_library(
     ],
 )
 
+wd_cc_library(
+    name = "mimetype",
+    srcs = ["mimetype.c++"],
+    hdrs = ["mimetype.h"],
+    implementation_deps = [
+        ":string-buffer",
+    ],
+    visibility = ["//visibility:public"],
+    deps = [
+        "//src/workerd/jsg:memory-tracker",
+        "@capnp-cpp//src/kj",
+        "@capnp-cpp//src/kj:kj-async",
+    ],
+)
+
+wd_cc_library(
+    name = "strings",
+    srcs = [],
```

Comment:
Late to this, but two minor potential improvements:
- Empty arguments like srcs and deps here are not needed since cc_library and by extension wd_cc_library supports defaut arguments
- If we apply the principles of IWYU to bazel, we'd want all bazel targets to be "self-sufficient", that is to depend directly on all targets they use. Then the strings target should depend on libkj, otherwise parsing the header would fail. This is a pretty small issue though, almost every source file will be including KJ in some capacity anyway.